src/test/java/com/launchdarkly/sdk/server/integrations/DynamoDbBigSegmentStoreImplTest.java

```diff
@@ -0,0 +1,98 @@
+package com.launchdarkly.sdk.server.integrations;
+
+import static com.launchdarkly.sdk.server.integrations.CollectionHelpers.mapOf;
+import static com.launchdarkly.sdk.server.integrations.DynamoDbStoreImplBase.PARTITION_KEY;
+import static com.launchdarkly.sdk.server.integrations.DynamoDbStoreImplBase.SORT_KEY;
+import static com.launchdarkly.sdk.server.integrations.TestUtils.TABLE_NAME;
+import static com.launchdarkly.sdk.server.integrations.TestUtils.baseBuilder;
+import static com.launchdarkly.sdk.server.integrations.TestUtils.clearEverything;
+import static com.launchdarkly.sdk.server.integrations.TestUtils.createTableIfNecessary;
+import static com.launchdarkly.sdk.server.integrations.TestUtils.createTestClient;
+
+import com.launchdarkly.sdk.server.interfaces.BigSegmentStoreFactory;
+import com.launchdarkly.sdk.server.interfaces.BigSegmentStoreTypes;
+
+import org.junit.BeforeClass;
+
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.PutItemRequest;
+import software.amazon.awssdk.services.dynamodb.model.UpdateItemRequest;
+
+/**
+ * Runs the standard database Big Segment store test suite that's defined in the Java SDK.
+ * <p>
+ * Note that you must be running a local DynamoDB instance on port 8000 to run these tests.
+ * The simplest way to do this is:
+ * <pre>
+ *     docker run -p 8000:8000 amazon/dynamodb-local
+ * </pre>
+ */
+@SuppressWarnings("javadoc")
+public class DynamoDbBigSegmentStoreImplTest extends BigSegmentStoreTestBase {
+
+  @BeforeClass
+  public static void setUpAll() {
+    createTableIfNecessary();
+  }
+
+  @Override
+  protected BigSegmentStoreFactory makeStore(String prefix) {
+    return baseBuilder().prefix(prefix);
+  }
+
+  @Override
+  protected void clearData(String prefix) {
+    clearEverything(prefix);
+  }
+
+  @Override
+  protected void setMetadata(String prefix, BigSegmentStoreTypes.StoreMetadata metadata) {
+    DynamoDbClient client = createTestClient();
+    String key = prefix + ":big_segments_metadata";
+    String syncTimeString = String.valueOf(metadata.getLastUpToDate());
+    client.putItem(PutItemRequest.builder()
+        .tableName(TABLE_NAME)
+        .item(
+            mapOf(
+                PARTITION_KEY, AttributeValue.builder().s(key).build(),
+                SORT_KEY, AttributeValue.builder().s(key).build(),
+                "synchronizedOn", AttributeValue.builder().n(syncTimeString).build()
```

Comment:
Could use `SYNC_TIME_ATTR` if it were package-private instead of private.

Reply:
I have a slight preference towards being explicit about the literal value in the tests for these attribute names since they need to be externally compatible with the `ld-relay` implementation, if only to make it slightly harder to accidentally modify the values during a refactor.

Reply:
That's fine.